guidelines/enhancement_template.md

```diff
@@ -218,7 +227,7 @@ enhancement:
 
 Upgrade expectations:
 - Each component should remain available for user requests and
-  workloads during upgrades. Any exception to this should be
+  workloads during upgrades. Ensure the components leverage best practices in handling voluntary disruption. Any exception to this should be
```

Comment:
I don't follow. All components should be leveraging best practices. What is this specifically getting at?

Reply:
This PR is about making sure that we state operational best practices explicitly, without getting into implementation details.  I could enumerate specifics here if you think it would be helpful, but it is really about following https://kubernetes.io/docs/concepts/workloads/pods/disruptions/  where applicable to the component.

Reply:
perhaps link it just so there's a reference in context?

Reply:
Will do!